q2_chemistree/tests/test_collate_fingerprint.py

```diff
@@ -8,17 +8,21 @@
 
 from unittest import TestCase, main
 from biom import load_table
+import pandas as pd
 import os
 from q2_chemistree import collate_fingerprint
 
 
-class fingerprintTests(TestCase):
+class FingerprintTests(TestCase):
     def setUp(self):
         THIS_DIR = os.path.dirname(os.path.abspath(__file__))
         self.featureTable = os.path.join(THIS_DIR,
                                          'data/features_formated.biom')
         self.emptycsi = os.path.join(THIS_DIR, 'data/emptycsi')
         self.goodcsi = os.path.join(THIS_DIR, 'data/goodcsi')
+        properties = os.path.join(THIS_DIR, 'data/molecular_properties.csv')
```

Comment:
You don't need to include two "molecular_properties.csv" files. You can just refer to the one that's installed in the package (like you do in `_collate_fingerprint.py`).